lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -408,6 +408,22 @@ namespace Js
         return JavascriptNumber::ToVarIntCheck(JavascriptConversion::ToInteger(value, scriptContext), scriptContext);
     }
 
+    Var JsBuiltInEngineInterfaceExtensionObject::EntryJsBuiltIn_Internal_IsCleanArray(RecyclableObject *function, CallInfo callInfo, ...)
```

Comment:
New internal "IsCleanArray" method returns true if an array is an ordinary array with an unmodified prototype and no getters/setters etc.